### PR TITLE
fix: ensure nav bar goes above all other components when scrolling

### PR DESCRIPTION
**Pull request by lynnshaoyu**
pointed out by kenzie in slack: https://uniswapteam.slack.com/archives/CKX7SR7E1/p1660760838891389

Before: 
Screen Shot 2022-09-06 at 2 49 52 PM

After:
Screen Shot 2022-09-06 at 2 49 27 PM

### CODE REVIEW

**Comment by aballerr** [comment]
This worked for me for both old and new navbar in browser and mobile. This also solves another issue with the widget when an error or setting state shows

bugged state
Screen Shot 2022-09-06 at 3 17 38 PM

With preview
Screen Shot 2022-09-06 at 3 12 27 PM

Screen Shot 2022-09-06 at 3 20 08 PM

File: src/pages/App.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ const HeaderWrapper = styled.div`
   justify-content: space-between;
   position: fixed;
   top: 0;
-  z-index: 2;
+  z-index: 9999;
```
Comment (aballerr) [inline src/pages/App.tsx:76]: maybe we should add an enum for this value and that here?
https://github.com/Uniswap/interface/blob/622581ee0aeaa6c72aa8345fe607810278ae5e0b/src/theme/index.tsx#L55

and refactor where this 9999 is used elsewhere in the app, I think it's just a couple of others

**Review by aballerr — APPROVED** [review]

**Comment by lynnshaoyu** [comment]
marking as draft to fix this issue jack pointed out, dropdown should be on top and not cutoff by nav bar: 
Screen Shot 2022-09-06 at 3 30 02 PM

File: src/pages/App.tsx (hunk 1 above)
Comment (lynnshaoyu) [reply src/pages/App.tsx]: sounds like a good idea done

**Comment by lynnshaoyu** [comment]
> marking as draft to fix this issue jack pointed out, dropdown should be on top and not cutoff by nav bar: Screen Shot 2022-09-06 at 3 30 02 PM

instead of upping the menu's z index i moved it down a bit so theres no overlap, similar to the chain and overflow menu dropdowns 

Screen Shot 2022-09-06 at 4 53 02 PM

**Pull request merged**